.github/workflows/renovate.yml

```diff
@@ -0,0 +1,20 @@
+name: Renovate
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '4 3 * * *'
+
+jobs:
+  renovate:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Self-hosted Renovate
+        uses: renovatebot/github-action@v39.1.1
```

Comment:
This will generate a lot of noise. Not too happy with that.

Reply:
yepp, they don't have a "rolling" branch :(